otb-luci/files/usr/lib/lua/luci/view/otb_overview.htm

```diff
@@ -72,6 +94,219 @@ <h2><%:Overview%></h2>
         return ret + seconds + ' seconds';
     }
 
+    /* start of otb-get-data section */
+    XHR.poll(15, '<%=url("admin/overthebox/data")%>', null, function(x,p) {
+        _otb_data = p;
+    });
+    function otb_get_data(...args) {
+        while(_otb_data.time == 0);
```

Comment:
What is this line for ?

Reply:
This was a try to avoid the use of a XHR poller, just forgoten to remove